doc/changelog/11-standard-library/19530-stdlib_repo.rst

```diff
@@ -0,0 +1,15 @@
+- **Changed:**
+  the development repository of Stdlib which is now
+  `https://github.com/coq-community/stdlib-test/ <https://github.com/coq-community/stdlib-test/>`_.  TODO update before merge
+  The recommended way to require
+  Stdlib modules is `From Stdlib Require {Import,Export,}
+  <LibraryModules>.`, except for `Byte` (use `From Stdlib.Strings` or
+  `From Stdlib.Init`), `Tactics` (use `From Stdlib.Program` or `From
+  Stdlib.Init`), `Tauto` (use `From Stdlib.micromega` of `From
+  Stdlib.Init`) and `Wf` (use `From Stdlib.Program` or `From
+  Stdlib.Init`). For backward compatibility with Coq <= 8.20, one can
```

Comment:
What's the point of these recommendations? If they are to remain, they should probably be explained better.

Reply:
Adapt to `From Stdlib` becoming "mandatory" (modulo the backward compatibility deprecation warning). While at it, we warn that the stdlib unfortunately has a few files with the same name that need a bit more qualification.

Reply:
Shouldn't these recommendations be part of the changelog for #19310 instead?

Reply:
#19310 only changes `From Coq` to `From Stdlib` but doesn't make it mandatory.

Reply:
OK, I see your point now. Maybe the two changelogs should then be merged, and some structure be added to it, because I'm afraid that currently, this one is quite hard to follow. (There's the change of prefix name, the prefix becoming mandatory, and the change in package structure---the change in repositories is an internal detail worth mentioning, but it isn't central to the user changelog IMHO.) Up to you.

Reply:
I agree, I'll try to merge the two.

Reply:
Did something change for users? People were already qualifying their requirements since it is a hard error otherwise. So, user files already contain either `[From Coq] Require Init.Byte` or `From Coq.Init Require Byte` or `Require Import Coq.Init.Byte`. In any case, users have to apply the exact same change as they would for any other requirement, i.e., add `From Stdlib` or change `Coq` to `Stdlib`. There is nothing specific to `Byte` here. So, while I don't mind the detailed paragraph, it should make it clearer that there is nothing new happening here (and if there is something new, it should state what).

Reply:
Indeed, that's nothing new, I tried to make it clear.